packages/contracts/contracts/Commitment.sol

```diff
@@ -0,0 +1,24 @@
+pragma solidity ^0.5.0;
+pragma experimental ABIEncoderV2;
+
+import "openzeppelin-solidity/contracts/math/Math.sol";
```

Comment:
Is this import used?

Reply:
Nope! Got rid of it because can't think of when it would be.

Reply:
I actually don't think the Commitment.sol contract is necessary anymore. Initially we were adding the verifyInclusion method on this and use it within Deposit.sol, but ended up just putting the method directly on the Deposit contract.

Reply:
Yep--but that was more for hackathon purposes, we would have needed to pull into an external contract eventually :)

Reply:
Yeah that's a good point. Probably just better to keep and iterate on it then.